source/Nevermore.Tests/Util/DataModificationQueryBuilderFixture.InsertSingleDocumentWithManyRelatedDocuments.approved.txt

```diff
@@ -1,11 +1,9 @@
 INSERT INTO [dbo].[TestDocumentTbl]  ([Id], [AColumn], [JSON])  VALUES 
 (@Id, @AColumn, @JSON)
-INSERT INTO [dbo].[RelatedDocument] ([Id], [Table], [RelatedDocumentId], [RelatedDocumentTable]) VALUES
-(@Id, 'TestDocumentTbl', @relateddocument_0, 'OtherTbl')
-,(@Id, 'TestDocumentTbl', @relateddocument_1, 'OtherTbl')
+INSERT INTO [dbo].[RelatedDocument] ([Id], [Table], [RelatedDocumentId], [RelatedDocumentTable])
+SELECT [Id], [Table], [RelatedDocumentId], [RelatedDocumentTable] FROM @0__relatedDocumentTableValuedParameter
 
 @Id=New-Id
 @JSON={}
 @AColumn=AValue
-@relateddocument_0=Rel-1
-@relateddocument_1=Rel-2
\ No newline at end of file
+@0__relatedDocumentTableValuedParameter=Nevermore.TableValuedParameter
```

Comment:
I haven't bothered to remove the prefix if there's only one - do we think this is important?